From: Surbhi Palande <surbhi.palande@canonical.com>
Date: Tue, 2 Jan 2024 10:23:29 +0100
Subject: Removed explicit printing of ':'

Removed the explicit printing of ':'. Expected to be a part of the prompt.
---
 src/plugins/splash/details/plugin.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/plugins/splash/details/plugin.c b/src/plugins/splash/details/plugin.c
index 9e150b5..d1543ed 100644
--- a/src/plugins/splash/details/plugin.c
+++ b/src/plugins/splash/details/plugin.c
@@ -399,10 +399,8 @@ display_password (ply_boot_splash_plugin_t *plugin,
                                 strlen (prompt));
         else
                 write_on_views (plugin,
-                                "Password",
-                                strlen ("Password"));
-
-        write_on_views (plugin, ":", strlen (":"));
+                                "Password:",
+                                strlen ("Password:"));
 
         for (i = 0; i < bullets; i++) {
                 write_on_views (plugin, "*", strlen ("*"));
